Documentation: corrections Beszel widget docs (#4282)
gethomepage/homepage: docs/widgets/services/beszel.md

description: Beszel Widget Configuration
---

Learn more about [Beszel](https://github.com/henrygd/beszel)

> The widget has two modes, a single system with detailed info if `systemId` is provided, or an overview of all systems if `systemId` is not provided. Note: `systemId` is not the system name. Find the correct ID in the list at `http://beszel.host.or.ip/_/#/collections` under the `id` field.

Reviewer: ```suggestion
The widget has two modes, a single system with detailed info if `systemId` is provided, or an overview of all systems if `systemId` is not provided. 

The `systemID` in the `id` field on the collections page of Beszel.
```
